### ISSUE TITLE
WADSmoosh: Needs its own TITLEPIC

### ISSUE THREAD

**Issue by Shockwave508** [nashmuhandes/WidePix#6]
By default, WADSmoosh uses a variant of the INTER_E4 graphic for its titlepic.

**Comment by madame-rachelle** [comment]
In my opinion this is something Wadsmoosh itself must control. Wadsmoosh has never had a static titlepic and it has indeed changed over the years. Wadsmoosh really needs to manage this one itself.

**Comment by nashmuhandes** [comment]
I agree. Sorry, this is something Wadsmoosh needs to handle as it is arbitrarily modifying the titlescreen. It is also an actively-maintained project.

WidePix's intention is to only enhance vanilla base games whose nature is no longer receiving any updates from their creators.

Matter of fact, now that I think back at it, I'm not sure if it was a good idea to even include Wadsmoosh support (no point doing it if it's only be done partially)... thoughts, @madame-rachelle ?

**Comment by madame-rachelle** [comment]
If you want to ditch wadsmoosh support, that is okay - however - the Wadsmoosh move did move some files out into "common"/"shared" areas and I think they should stay there, because Wadsmoosh might in the future not be the only game combiner. I think the original id Software graphics that are already moved should stay where they are - and be in the shared areas - because I think it should not be up to WidePix whether to provide the graphics or not, if such graphics can indeed be provided.

That being said though - keep in mind that Wadsmoosh is "officially" supported by GZDoom. While you are right that it is an active project - there isn't really much to say that support "shouldn't" be provided, more is saying that it "should" be.

**Comment by nashmuhandes** [comment]
Okay, we're not ditching Wadsmoosh support - we've just found a way to add support to it without providing and physically-duplicate files: https://github.com/nashmuhandes/WidePix/commit/ccf56490732e9cabcb7c182900681c16c14aebf2

That said, the issue about the title screens however - yeah, I think it's up to JPL to provide a bespoke Wadsmoosh title image. He of course has full permissions to take whatever he wants from WidePix to use a base, but - after some time has passed, and if there are still no developments in that area, I may consider contributing some custom titlescreen art for Wadsmoosh. We'll see.

At any rate, I'll leave this issue open for now until I can sit down to think about this further.

**Comment by madame-rachelle** [comment]
I will bring this to the attention of JPL. To be quite honest I liked Wadsmoosh's old title screen better. This might be a good opportunity for him to choose one from the newly created WidePix assets, anyhow, if he wants.

**Comment by JPLeBreton** [comment]
Hi, thanks for the heads up @madame-rachelle. At some point I will do a new WadSmoosh release with its own title image - not exactly sure what it'll be yet but it's good to know WidePix are an option. Thanks!
